### DISCUSSION TITLE
Unable to update remote Mac RustDesk version

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#9560] [General]
### Bug Description

I am unable to update RustDesk remotely on a Mac. The update operation must be performed by a local user because it requires stopping the service that allows unattended remote connections. Is there a way to remotely update RustDesk to a new version on a Mac?

### How to Reproduce

Try to remotly update RustDesk on a remote Mac

### Expected Behavior

Easy update of the version

### Operating system(s) on local side and remote side

All -> OSX

### RustDesk Version(s) on local side and remote side

1.3.1-> 1.3.0

### Screenshots

No screenshot is useful

**Comment by Poordeveloper** [comment]
Following https://github.com/rustdesk/rustdesk/discussions/7762, you can write a script to do this, though this does need some effort, chatgpt can help you a lot I believe.
